docs/section22/Rule22-37.md

```diff
@@ -0,0 +1,52 @@
+
+# CHW&CW - Rule 22-37  
+
+**Schema Version:** 0.0.16  
+**Mandatory Rule:** True  
+**Rule ID:** 22-37  
+**Rule Description:** The baseline chiller efficiencies shall be modeled at the minimum efficiency levels for part load, in accordance with Tables G3.5.3.  
+**Rule Assertion:** B-RMR = expected value  
+**Appendix G Section:** Section 22 CHW&CW Loop  
+**90.1 Section Reference:** Section G3.1.2.1 Equipment Efficiencies  
+**Data Lookup:** Table G3.5.3  
+**Evaluation Context:** Building  
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system that is Type-7, 8, 11.1, 11.2, 12, 13, 7b, 8b, 11b, 12b, 13b.
+
+**Function Calls:**  
+
+1. get_baseline_system_types()
+2. data_lookup()
+
+**Applicability Checks:**  
+
+- Get B-RMR system types: `baseline_hvac_system_dict = get_baseline_system_types(B-RMR)`
+
+  - Check if B-RMR is modeled with at least one air-side system that is Type-7, 8, 11.1, 11.2, 12, 13, 7b, 8b, 11b, 12b, 13b, i.e. with air-side system served by chiller(s), continue to the next applicability check: `if any(sys_type in baseline_hvac_system_dict.keys() for sys_type in ["SYS-7", "SYS-8", "SYS-11.1", "SYS-11.2", "SYS-12", "SYS-13", "SYS-7B", "SYS-8B", "SYS-11B", "SYS-12B", "SYS-13B"]): CHECK_RULE_LOGIC`
+
+  - Else, rule is not applicable to B-RMR: `else: RULE_NOT_APPLICABLE`
+
+## Rule Logic:  
+
+- For each chiller in B_RMR: `for chiller_b in B_RMR.RulesetModelInstance.chillers:`
+
+  - Get chiller rated capacity: `rated_capacity_b = chiller_b.rated_capacity`
+
+    - If chiller rated capacity is less than 150Ton, set size category to < 150TONS: `if rated_capacity_b < 150 * CONVERSION(TON_TO_BTUH): size_category = "< 150TONS"`
+
+    - Else if chiller rated capacity is 150Ton or more and is less than 300Ton, set size category to >= 150TONS and < 300TONS: `else if rated_capacity_b < 300 * CONVERSION(TON_TO_BTUH): size_category = ">= 150TONS AND < 300TONS"`
+
+    - Else if chiller rated capacity is 300Ton or more and is less than 600Ton, set size category to >= 300TONS and < 600TONS": `else if rated_capacity_b < 600 * CONVERSION(TON_TO_BTUH): size_category = ">= 300TONS AND < 600TONS"`
+
+    - Else, set size category to > 600TONS: `else: size_category = "> 600TONS"`
+
+    - Get baseline chiller efficiency requirements: `kW_ton_part_load_b = data_lookup(table_G3_5_3, chiller_b.compressor_type, size_category, "IPLV")`
+
+      **Rule Assertion - Component:**
+
+      - Case 1: For each chiller, if chiller part-load efficiency are modeled at the minimum efficiency levels in accordance with Tables G3.5.3 `if ( chiller_b.part_load_efficiency == part_load_efficiency * CONVERSION(KW_PER_TON_TO_COP) ) AND ( chiller_b.part_load_efficiency_metric == "INTEGRATED_PART_LOAD_VALUE" ): PASS`
+
```

Comment:
@xingcx i believe the statement chiller_b.part_load_efficiency == part_load_efficiency * CONVERSION(KW_PER_TON_TO_COP) ) 
needs to be revised to 
chiller_b.part_load_efficiency == **kW_ton_part_load_b** * CONVERSION(KW_PER_TON_TO_COP) )
correct?

Reply:
That is correct. RDS is modified. And Maria's comment for future reference, "We discussed that RCT will convert metric values to IP as it parses the RMD files. We need to make sure that chiller efficiencies stay as COPs as the values are parsed."